msyncd/gschemas/com.meego.msyncd.gschema.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<schemalist>
+  <schema id="com.meego.msyncd" path="/com/meego/msyncd/">
+    <key name="allow-scheduled-sync-over-gsm" type="b">
+      <summary>Scheudled Sync over GSM</summary>
```

Comment:
- Typo
- Might be better to use word "cellular" instead of gsm unless you want to control sync over one type of 2G connection.
- Not sure if dependency to QGSettings is good here. Any option to use mlite/MGConfItem which nowadays wraps dconf, or then dconf directly? What's the upstream for QGSettings anyway?